test/chunksaver.go

```diff
@@ -91,7 +94,7 @@ func openLogChunkConsumingFunc(outputPath string, outputConfig bconfig.LogOutput
 				chunkFilePath := fmt.Sprintf(outputPath, chunk.ID)
 				tmp := &bytes.Buffer{}
 				tmp.WriteString("[\n")
-				if _, err := outputConfig.DumpRecordsAsJSON(chunk, []byte(",\n"), true, tmp); err != nil {
+				if _, err := fluentdforward.ConvertMsgpackToJSON(chunk, []byte(",\n"), true, tmp); err != nil {
```

Comment:
We might end up needing to support multiple outputs from inside the test-pipeline.

What do you think of adding a reference to outputConfig in LogChunk itself? i.e. each of generated chunk contains a reference to the output setup and thus can be used to covert the binary content to JSON? Though the only use of that would be for testing, not in production.

Reply:
Yea, but don't we serialize and buffer this LogChunk on disk at some point? In which case the reference will probably be dereferenced during marshalling and the stored chunk will be full of config data it doesn't even use? 
Can't we just store the output type in LogChunk cause that's the only thing we really need?